src/qml/components/BusyControl.qml

```diff
@@ -0,0 +1,109 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+
+import "../theme"
+
+Item {
```

Comment:
File is a 75% copy-paste of existing loader. Not acceptable

Reply:
For C++ ? True, but not for qml. Moreover, the functionality of different loaders is not compatible, although it seems that the code is very similar. If we try to remove duplicate code, we will only spoil the source loader code.

Reply:
fix

Reply:
No